src/pages/contributors/jesse-glick.adoc

```diff
@@ -0,0 +1,117 @@
+= Jesse Glick
+:page-name: Jesse Glick
+:page-linkedin:
+:page-twitter: 
+:page-github: jglick
+:page-email:
+:page-image: avatar/jesse-glick.jpeg
+:page-pronouns:
+:page-location: Raleigh, North Carolina, USA
+:page-firstcommit: 2009
+:page-datepublished: 2024-11-05
+:page-featured: true
+:page-intro: Jesse Glick is a software engineer, currently residing near Raleigh, North Carolina, and has been involved with the Jenkins project for over a decade. Prior to contributing to Jenkins, Jesse worked as an engineer at Sun Microsystems. After incorporating Jenkins into his work at Sun, albeit in a small way, there wasn't much more to do with the project. However, that all changed when Kohsuke reached out to Jesse to find out if he would be interested in working on Jenkins full-time. Since joining CloudBees in 2012, Jesse has been an integral part of Pipeline development, and incorporating many functions that are now fundamental to current Pipeline operation. Jesse works constantly on improving the Pipeline experience, and shares his insights on how to improve not only the Jenkins project but the open-source workflow as a whole.
+
+== What is your background prior to contributing to Jenkins?
+
+I worked at Sun Microsystems for quite a long time, working on the NetBeans IDE.
+We had some CI processes, such as Cruise Control, as part of that.
+While there, I heard about Jenkins (then Hudson) in its infancy.
+I knew of Kohsuke and his work on Hudson and thought it might be something we could start using.
+At the time, we had been using CVS for version control, which Hudson had incorporated to some degree.
+
+We migrated from CVS to Mercurial SCM, which I also wound up working on in Hudson.
+It was just enough to get the features we needed for our usage patterns and we didn't pay more attention to it beyond that.
+I had also done some work on the tool system where you can have tool installations and automatic downloads.
+That was mostly driven by Andrew Bayer, another long-time Jenkins contributor.
+After the Jenkins fork was created, Kohsuke contacted me and asked if I wanted to come join him at the company where he was working on Jenkins.
+This was in 2012 when I joined CloudBees, and I have been focused on Jenkins ever since.
```

Comment:
```suggestion
I worked at Sun Microsystems for quite a long time on the NetBeans IDE.
We had some CI processes, such as Cruise Control, as part of that.
While there, I heard about Jenkins (then Hudson) in its infancy.
I knew of Kohsuke and his work on Hudson and thought it might be something we could start using.
At the time, we had been using CVS for version control, which Hudson had incorporated to some degree.

We migrated from CVS to Mercurial SCM, which I also wound up working on in Hudson.
It was just enough to get the features we needed for our usage patterns and we didn't pay more attention to it beyond that.
I had also done some work on the tool system where you can have tool installations and automatic downloads.
That was mostly driven by Andrew Bayer, another long-time Jenkins contributor.
After the Jenkins fork was created, Kohsuke contacted me and asked if I wanted to come join him at the company where he was working on Jenkins.
This was in 2012 when I joined CloudBees, and I have been focused on Jenkins ever since.
```